index.html

```diff
@@ -0,0 +1,1209 @@
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>The evaluation of an authorization is concerned with finding Authorizations that match the required parameters of an operation (<cite><a href="#authorization-conformance" rel="rdfs:seeAlso">Authorization Conformance</a></cite>). Evaluation stops when all access permission requests have been granted by one or more Authorizations. Authorizations that do not match a required access permission have no effect on the outcome of the evaluation. Access is granted when conforming Authorizations are matched, otherwise access is denied.</p>
+
+                  <p>As per determining the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> of a resource, an Authorization can be evaluated explicitly (via <code>acl:accessTo</code>) or inherited (via <code>acl:default</code>) against a particular request.</p>
+
+                  <p>When a request requires an access mode (<code>acl:mode</code>) which is a limitation of another access mode, then access is granted if either mode is allowed by an Authorization. For example, when a request requires <code>acl:Append</code>, then access will be granted to agents having <code>acl:Write</code>.</p>
+
+                  <p>The presence of the <code>acl:origin</code> property and its value is taken into account in the evaluation only when the HTTP request includes the <code>Origin</code> header.</p>
+
+                  <section id="reading-writing-resources" inlist="" rel="schema:hasPart" resource="#reading-writing-resources">
+                    <h4 property="schema:name">Reading and Writing Resources</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Working alongside <cite><a href="#http-interaction" rel="rdfs:seeAlso">HTTP Interactions</a></cite>, WAC enables a standard set of operations which can be applied to different resource types for a given request based on the set of access modes which control those operations:</p>
+
+                      <p>Generally: read operations attempt to confirm the existence of a resource or to view the contents of a resource; write operations attempt to create, delete, or modify resources; append operations attempt to create resources or add information to existing resources; and control operations attempt to view, create, delete, or modify ACL resources.</p>
+
+                      <p>As container resources and member resources are hierarchically organised, requests to perform operations on resources are in the context of the applicable container (<cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite>).</p>
+
+                      <p>When an operation requests to read a resource, the server MUST match an Authorization allowing the <code>acl:Read</code> access privilege on the resource.</p>
```

Comment:
Maybe note that it can be possible to obtain some information about existence of a resource without read access sometimes. For instance if I only have `accessTo Write` and `default Write` on `/` then I am allowed to `PUT /secret If-None-Match: *` and then `DELETE /secret` if the PUT succeeded. And then I will have "read" the information that /secret did not exist.

There have been long debates about whether this should be regulated more tightly, and I think we concluded to let it be. In any case, worth mentioning it, I think.

Reply:
Perhaps something general along the lines of "users should not rely on the confidentiality of URIs or whether a certain resource exists or not for their security requirements"

Reply:
(Not sure I see the significance of DELETE in your example.)

Wouldn't this be part of mapping incoming request to required operation and Authorization? For instance, server may require only Write for vanilla PUT. When If-None-Match: * is included in the request, it can additionally require Read.

Feedback from implementations would be pretty useful here.

Reply:
Right, yes! Please mark as resolved.